apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/completionWorks.js

```diff
@@ -28,7 +24,7 @@ const buildExportWordFile = async ({ task, documentData, locale, timeZone }) =>
 
     return {
         stream,
-        filename: `completion_works_ticket_${ticket.id}_${formatDate(undefined, timeZone, 'DD_MM_YYYY')}.docx`,
+        filename: `completion_works_ticket_${ticket.id}_${formatDateToTimezone(undefined, timeZone, 'DD_MM_YYYY')}.docx`,
```

Comment:
Why do you need to create this function?

Reply:
This function was there before. I just changed the name from formatDate to formatDateToTimezone. I use this function in several places

Reply:
But it is in diff